source/codegen/metadata/nidcpower/config.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DCPower API metadata version 20.7.0d8
+config = {
+    'api_version': '20.7.0d8',
+    'c_header': 'nidcpower.h',
+    'c_function_prefix': 'niDCPower_',
+    'service_class_prefix': 'NiDCPower',
+    'java_package': 'com.ni.grpc.nidcpower',
+    'csharp_namespace': 'NationalInstruments.Grpc.DCPower',
```

Comment:
@ganeshnie , I will change it to the convention used in scope. Scope has the line **'java_package': 'com.ni.grpc.scope'**, so accordingly for dcpower it would become **'java_package': 'com.ni.grpc.dcpower'**